proto/eventlog.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package eventlog;
+
+option go_package = "proto;eventlog";
+
+message GetEventLogChunkRequest {
+  // The request context.
+  context.RequestContext request_context = 1;
+
+  // The invocation identifier itself.
+  string invocation_id = 2;
+
+  message OptionalChunklID {
```

Comment:
blah blah proto2, proto3, etc. (we talked about this on vc)

can you just make chunk_id a top level-string and remove the optional wrapper?